samples/mini-games/settings.gradle.kts

```diff
@@ -0,0 +1,13 @@
+includeBuild("../../") {
+    dependencySubstitution {
+        substitute(module("com.utopia-rise:godot-gradle-plugin")).with(project(":godot-gradle-plugin"))
+    }
+}
+
+pluginManagement {
+    resolutionStrategy.eachPlugin {
+        when (requested.id.id) {
+            "com.utopia-rise.godot-kotlin" -> useModule("com.utopia-rise:godot-gradle-plugin:0.1.0")
```

Comment:
Also here should the version not go into the versions file in the buildSrc?

Reply:
It's not accessible here, the `0.1.0` is here so that gradle won't complain. Basically it is just a placeholder value to make gradle happy.

Reply:
I see. Ok then ^^